### ISSUE TITLE
Warn users if private_key_path is not set

### ISSUE THREAD

**Issue by Temikus** [mitchellh/vagrant-google#49]
If user doesn't specify a private key in the Vagrantfile, as in:
```
  override.ssh.username = "temikus"
  override.ssh.private_key_path = "~/.ssh/id_rsa"
```
The machine tries to start with a default Vagrant insecure private key.
However, if it is not added to the metadata and we just run a default Vagrantfie, it will start the instance, but will hang on SSH:
```
==> default: Waiting for instance to become "ready"...
==> default: Machine is booted and ready for use!
==> default: Waiting for SSH to become available...
```
We need to:
- Update the docs about adding vagrant key to metadata.
- [If possible] warn users if private_key_path is not overriden.

**Comment by Temikus** [comment]
/CC @erjohnso 
This can now be closed with merge of #64

**Comment by erjohnso** [comment]
Great, thanks @Temikus !
